ledger/participant-integration-api/src/main/scala/platform/apiserver/configuration/LedgerConfigurationSubscriptionFromIndex.scala

```diff
@@ -130,7 +136,8 @@ private[apiserver] final class LedgerConfigurationSubscriptionFromIndex(
 
     override def ready: Future[Unit] = readyPromise.future
 
-    override def latestConfiguration(): Option[Configuration] = latestConfigurationState.get._2
+    override def latestConfiguration(): Option[Configuration] =
```

Comment:
```suggestion
    override def latestConfiguration: Option[Configuration] =
```

Or was it intentional?

Reply:
Yup, very intentional. I don't like omitting parentheses for referentially-opaque functions.